### DISCUSSION TITLE
v1/en/docs/widgets/funding/

### DISCUSSION THREAD

**Discussion by giscus** [razonyang/hugo-theme-bootstrap#955] [General]
# v1/en/docs/widgets/funding/

The funding widget, AKA buy me a coffee widget, gives the readers a way to support a creator.

https://hbs.razonyang.com/v1/en/docs/widgets/funding/

**Comment by TheFriendlyCoder** [comment]
Do you have any plans to add support for BMC (buymeacoffee) to your funding widget?

**Reply by razonyang** [reply]
Added it in [`v1.1.0`](https://github.com/razonyang/hugo-theme-bootstrap/releases/tag/v1.1.0).
